<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab1">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <attribute name="title">
      <string>쪽</string>
     </attribute>
     <widget class="QGroupBox" name="group1_tab1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>591</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>계좌정보</string>
      </property>
      <widget class="QTableWidget" name="tableAccount1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>571</width>
         <height>81</height>
        </rect>
       </property>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>23</number>
       </attribute>
       <column>
        <property name="text">
         <string>예수금</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총매입</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총평가</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총손익</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총수익률</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>추정자산</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group2_tab1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>591</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>보유종목 정보</string>
      </property>
      <widget class="QTableWidget" name="tableAccount2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>571</width>
         <height>201</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>보유량</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>매입가</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>현재가</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>평가손익</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수익률</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboAccount">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>421</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushAccount">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAccount">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>◆ 계좌선택:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_select">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>종목선택:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_date">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>기준일자:</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>601</width>
        <height>291</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab2_1">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTableView" name="OHLCChart">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>571</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab2_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QTableView" name="OHLCDataFrame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>571</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>쪽</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QRadioButton" name="radio_Exchange">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>장내주식</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_KOTC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>K-OTC</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_KOSDAQ">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>KOSDAQ</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="marketSelect">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>시장선택</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>321</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>종목선택</string>
      </property>
      <widget class="QPushButton" name="stockSearch">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>수동매매</string>
      </property>
     </widget>
     <zorder>groupBox_2</zorder>
     <zorder>marketSelect</zorder>
     <zorder>comboBox</zorder>
     <zorder>label_select</zorder>
     <zorder>dateEdit</zorder>
     <zorder>label_date</zorder>
     <zorder>tabWidget2</zorder>
     <zorder>radio_Exchange</zorder>
     <zorder>radio_KOTC</zorder>
     <zorder>radio_KOSDAQ</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>최소 투자 단위</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>최대 투자 단위</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>지연 보상 임계치</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Learning Rate</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Epoch</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Epsilon</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>310</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Parameter Setting</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>모델 시작일</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>모델 종료일</string>
      </property>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>spinBox</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>spinBox_2</zorder>
     <zorder>doubleSpinBox</zorder>
     <zorder>doubleSpinBox_2</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>spinBox_3</zorder>
     <zorder>doubleSpinBox_3</zorder>
     <zorder>dateEdit_2</zorder>
     <zorder>dateEdit_3</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>쪽</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
